flink-streaming-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/testutils/MockRuntimeContext.java

```diff
@@ -148,6 +149,11 @@ public Histogram getHistogram(String name) {
 	}
 
 	@Override
+	public LongHistogram getLongHistogram(String name) {
```

Comment:
I think we should try and not have utility methods for each accumulator type in the runtime context - it becomes a lot otherwise. The methods for `getHistogram()` etc are also marked as public evolving, because they may possibly be removed in the future.